packages/scheduled-tasks/src/index.ts

```diff
@@ -0,0 +1,20 @@
+import type { ScheduledTaskHandler } from './lib/ScheduledTaskHandler';
+import type { ScheduledTasksOptions } from './lib/types';
+
+export * from './lib/ScheduledTaskHandler';
+export * from './lib/strategies';
+export * from './lib/structures/ScheduledTask';
+export * from './lib/structures/ScheduledTaskStore';
+export * from './lib/types';
+
+declare module '@sapphire/pieces' {
+	interface Container {
+		tasks: ScheduledTaskHandler;
+	}
+}
+
+declare module 'discord.js' {
+	export interface ClientOptions {
+		tasks?: ScheduledTasksOptions;
```

Comment:
TODO: @kyranet @Favna move these plugin options to a `ClientOptions#pluginOptions#<whateverTheyChoose>`